docs/team/lfrostbytee.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: page
+title: Ong Wee Marcus's Project Portfolio Page
+---
+
+### Project: Duke The Market
```

Comment:
Are we calling it Duke-The-Market?

Reply:
Let's settle on "Duke The Market" as we have previously submitted that as the name of our product. Our product name has also been set and can be seen on the [CS2103 website](https://nus-cs2103-ay2223s1.github.io/website/admin/teamList.html#cs2103-f09-2-duke-the-market)

Reply:
Sure, though there may be some places with the incorrect name "Duke-The-Market" which we need to change.
E.g. The [PR](https://github.com/nus-cs2103-AY2223S1/tp/pull/84) from our team repo to the upstream CS2103 repo.

Reply:
Yup, thanks for pointing that out. So far besides the PR, I haven't noticed anywhere else that uses "Duke-The-Market" yet. In the meantime, I have changed the title of the [PR](https://github.com/nus-cs2103-AY2223S1/tp/pull/84) to "Duke The Market"